proxies/go/handler_client.go

```diff
@@ -58,9 +59,9 @@ func clientHandler(w http.ResponseWriter, r *http.Request) {
 		ConfigPollingIntervalMS: time.Duration(reqBody.Options.ConfigPollingIntervalMS * 1000000),
 		EventFlushIntervalMS:    time.Duration(reqBody.Options.EventFlushIntervalMS * 1000000),
 		EnableCloudBucketing:    reqBody.EnableCloudBucketing,
-		OnInitializedChannel:    onInitializedChannel,
+		ClientEventHandler:      onInitializedChannel,
```

Comment:
should the `onInitializedChannel` name change?

Reply:
in this case we don't actually care about the name since we don't consume events from it - but it could be a good idea.